test/unit/mock/mock_http_client.js

```diff
@@ -1052,7 +1056,8 @@ function buildRequestOutputMappings(clientInfo)
           json:
             {
               disableOfflineChunks: false,
-              sqlText: 'select \'too many concurrent queries\';'
+              sqlText: 'select \'too many concurrent queries\';',
+              queryContextDTO: { entries: [] }
```

Comment:
do we need to add this query context to add reponses? are we not able to work where query context is null/undefined?